src/backend/install_nextclade.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+eval "$($HOME/miniconda/bin/conda shell.bash hook)"
```

Comment:
Any reason we're using conda to install nextclade instead of using their cli-download option?
https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli.html#download-from-command-line

Reply:
i thought the conda install was cleaner than using curl, plus we already have to install miniconda for pangolin so might as well

Reply:
I'd really prefer to avoid using conda as much as possible. If we get into a situation where we just *have* to manage conda environments due to conflicting package requirements, we can deal with it then, but in the meantime the curl installation is pretty clean (it's how we install miniconda already 😢 )